src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+
+  --link-main-color: #000;
+  --link-active-color: #00acdc;
+}
+
 body {
+  margin: 0 auto;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header_logo {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.nav__list {
   margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Instead of styling '.nav__list' directly, consider adding a class to the 'ul' element and style that class. This makes the CSS more reusable and keeps it consistent with the rest of your styling approach.